src/main/resources/kn.json

```diff
@@ -1,30 +1,30 @@
 {
   "tools": {
     "kn": {
-      "version": "1.5.0",
+      "version": "20220823-local-6d86bf75",
       "versionCmd": "version",
-      "versionExtractRegExp": "Version:\\s*v(\\d+[\\.\\d+]*)",
-      "versionMatchRegExpr": "1.5.0",
+      "versionExtractRegExp": "Version:\\s*v(\\w+-\\w+-\\w+)",
+      "versionMatchRegExpr": "20220823-local-6d86bf75",
```

Comment:
I don't understand how it worked before because 1.5.0 already uses this version identifier so don't understand how it worked before.

Reply:
It always worked because it also checks if the path exists
```
if (!areCompatible(version, tool.getVersionMatchRegExpr())) {
            Path path = Paths.get(tool.getBaseDir().replace("$HOME", CommonConstants.HOME_FOLDER), "cache", tool.getVersion(), command);
            final String cmd = path.toString();
            if (!Files.exists(path)) {
                downloadInBackground(toolName, platform, path, cmd, tool, version, result);
            } else {
                result.complete(cmd);
            }
```